src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,69 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.10.0dev1 (08/03/2018)
+-----------------------
+
+New features
+~~~~~~~~~~~~
+
+* Add --output-dir flag to ScalaFmt task (#6134)
+  `PR #6134 <https://github.com/pantsbuild/pants/pull/6134>`_
+
+Bugfixes
+~~~~~~~~
+
+* Fix a deadlock in local process execution. (#6292)
+  `PR #6292 <https://github.com/pantsbuild/pants/pull/6292>`_
+
+* When python target compatibility is not set, use interpreter constraints. (#6284)
```

Comment:
I'd say this counts as a new feature.

Reply:
It's labeled as a bug by @jsirois 

https://github.com/pantsbuild/pants/issues/5082